src/content/docs/ru/tutorial/4-layouts/3.mdx

```diff
@@ -0,0 +1,139 @@
+      <p>{frontmatter.pubDate.slice(0,10)}</p>
+      <p><em>{frontmatter.description}</em></p>
+      <p>Написал: {frontmatter.author}</p>
+      <img src={frontmatter.image.url} width="300" alt={frontmatter.image.alt} />
+      <slot />
+    </BaseLayout>
+    ```
+
+2. Проверьте предварительный просмотр в браузере по адресу `localhost:3000/posts/post-1`. Теперь вы должны видеть содержимое, отрендеренное:
+
+   - Вашим **основным макетом страницы**, включая стили, ссылки для навигации и социальные подвалы.
+   - Вашим **макетом поста в блоге**, включая свойства frontmatter, такие как описание, дата, заголовок и изображение
+   - Вашим **отдельным содержимым поста Markdown**, включая только текст, написанный в этом посте
+
+3. Обратите внимание, что заголовок вашей страницы теперь отображается дважды, в каждом макете.
+
+    Удалите строку, которая отображает заголовок вашей страницы из `MarkdownPostLayout.astro`:
+
+    ```astro title="src/layouts/MarkdownPostLayout.astro" del={2}
+    <BaseLayout pageTitle={frontmatter.title}>
+      <h1>{frontmatter.title}</h1>
+      <p>{frontmatter.pubDate.slice(0,10)}</p>
+      <p><em>{frontmatter.description}</em></p>
+      <p>Написал: {frontmatter.author}</p>
+      <img src={frontmatter.image.url} width="300" alt={frontmatter.image.alt} />
+      <slot />
+    </BaseLayout>
+    ```
+
+5. Снова проверьте предварительный просмотр в браузере по адресу `localhost:3000/posts/post-1` и убедитесь, что эта строка больше не отображается и что ваш заголовок отображается только один раз. Внесите все необходимые корректировки, чтобы убедиться, что у вас нет дублирующегося контента.
+
+  Убедитесь, что:
+
+    - Каждый пост в блоге показывает один и тот же шаблон страницы, и никакого контента не пропущено. (Если в одном из ваших постов в блоге отсутствует контент, проверьте его свойства frontmatter.)
+
+    - На странице не дублируется контент. (Если что-то отображается дважды, обязательно удалите это из `MarkdownPostLayout.astro`.)
+
+    Если вы хотите настроить шаблон вашей страницы, вы можете это сделать.
+
+
+
+
+<Box icon="question-mark">
+
+### Проверьте свои знания
+
+1. Это позволяет вам вкладывать один макет в другой и использовать преимущества работы с модульными частями.
+
+<MultipleChoice>
+  <Option>
+    непрерывное развертывание
+  </Option>
+  <Option>
+    адаптивный дизайн
+  </Option>
+  <Option isCorrect>
+    компонентный дизайн
+  </Option>
+</MultipleChoice>
+
+2. Несколько макетов особенно полезны для проектов, которые содержат страницы Markdown, как например…
+
+<MultipleChoice>
+  <Option isCorrect>
+    блог
+  </Option>
+  <Option>
+    панель приборов
+  </Option>
+  <Option>
+    чат-приложение
+  </Option>
+</MultipleChoice>
+
+3. Кто из них предоставляет шаблонирование для всех ваших страниц?
+
+<MultipleChoice>
+  <Option>
+    `index.astro`
+  </Option>
+  <Option isCorrect>
+    `BaseLayout.astro`
+  </Option>
+  <Option>
+    `post-1.md`
+  </Option>
+</MultipleChoice>
```

Comment:
```suggestion
    <MultipleChoice>
        <Option>
            непрерывное развертывание
        </Option>
        <Option>
            адаптивный дизайн
        </Option>
        <Option isCorrect>
            компонентный дизайн
        </Option>
    </MultipleChoice>

2. Несколько макетов особенно полезны для проектов, которые содержат страницы Markdown, как например…

    <MultipleChoice>
        <Option isCorrect>
            блог
        </Option>
        <Option>
        панель приборов
        </Option>
        <Option>
            чат-приложение
        </Option>
    </MultipleChoice>

3. Кто из них предоставляет шаблонирование для всех ваших страниц?

    <MultipleChoice>
        <Option>
            `index.astro`
        </Option>
        <Option isCorrect>
            `BaseLayout.astro`
        </Option>
        <Option>
            `post-1.md`
        </Option>
    </MultipleChoice>
```
The `<MultipleChoice>` component needs to be indented when inside an ordered list to render correctly